lib/apr/views/commerce/commerce_transaction_slack_view.ex

```diff
@@ -140,38 +140,54 @@ defmodule Apr.Views.CommerceTransactionSlackView do
 
   defp seller_admin(_), do: []
 
-  defp stripe_fields(external_id, external_type) do
-    with {:ok, pi} <- @payments.payment_info(external_id, external_type) do
-      [
-        %{
-          title: "Risk Level",
-          value: pi.charge_data.risk_level,
-          short: true
-        },
-        %{
-          title: "Card Country",
-          value: pi.card_country,
-          short: true
-        },
-        %{
-          title: "Billing State",
-          value: pi.billing_state,
-          short: true
-        },
-        %{
-          title: "CVC Check  #{format_check(pi.cvc_check)}",
-          short: true
-        },
-        %{
-          title: "ZIP Check  #{format_check(pi.zip_check)}",
-          short: true
-        },
-        %{
-          title: "Liability Shift #{format_boolean(pi.charge_data.liability_shift)}",
-          short: true
-        }
-      ]
-    else
+  defp stripe_fields(external_id, external_type, payment_method) do
+    case payment_method do
```

Comment:
#non-blocking. Not an elixir expert but potentially we can pattern match on the `payment_method` argument and skip this level of switch cases.

Reply:
hmm not sure I understand what you mean @starsirius - do you mean having a single `payment_info/3` method instead of separate `payment_info/2` and `payment_info_ach/2` methods? Something else?

Reply:
I wonder if we can do something like and make the structure a bit flat.

```elixir
defp stripe_fields(external_id, external_type, "credit card") do
  # ...
end

defp stripe_fields(external_id, external_type, "us_bank_account") do
  # ...
end
```